webviz_subsurface/plugins/_parameter_analysis/models/parameters_model.py

```diff
@@ -206,39 +206,94 @@ def make_grouped_plot(
                         ),
                         20,
                     ),
+                    facet_row_spacing=max((0.08 - (0.00071 * len(parameters))), 0.03),
```

Comment:
This feels a bit hacky. But yes, I know from experience that this scaling is hard in Plotly / Dash. 
Have you tested on different screen sizes and resolutions?

Reply:
Yes, agree, this is a bit hacky... but needed when you have many parameters plotted at the same time (otherwise the plots are squeezed and the spacing around them are large). 
From Plotly's documentation they state that this is in reference to the paper coordinates of a plot, so it should work on different screen resolutions. 

Ideally I would like to have this number adjusted by Plotly, they state themselves that it should be modified based on how many plots a user have: 

> The defaults work well with 1-4 rows or columns at the default figure size with the default font size, but need to be reduced to around 0.01 for very large figures or figures with many rows or columns. Conversely, if activating tick labels on all facets, the spacing will need to be increased.

But I guess for now this is the solution at hand.